packages/gitbook/src/components/DocumentView/Expandable/Details.tsx

```diff
@@ -0,0 +1,84 @@
+'use client';
+
+import React from 'react';
+
+import { useHash } from '@/components/hooks';
+import { ClassValue, tcls } from '@/lib/tailwind';
+
+/**
+ * Details component rendered on client so it can expand dependent on url hash changes.
+ */
+export function Details(props: {
+    children: React.ReactNode;
+    id: string;
+    contentIds?: string[];
+    open?: boolean;
+    className?: ClassValue;
+}) {
+    const { children, id, open, className } = props;
+
+    const detailsRef = React.useRef<HTMLDetailsElement>(null);
+
+    const [anchorElement, setAnchorElement] = React.useState<Element | null | undefined>();
```

Comment:
Useless but now you can have only `const [open, setOpen] = React.useState(false);`